src/python/pants/bin/pants_runner.py

```diff
@@ -57,7 +65,10 @@ def run(self):
     for message_regexp in global_bootstrap_options.ignore_pants_warnings:
       warnings.filterwarnings(action='ignore', message=message_regexp)
 
-    if global_bootstrap_options.enable_pantsd:
+    # TODO For kill-pantsd and clean-all, we cannot run them in the daemon itself, because
```

Comment:
There are #7205 (issue) and #7206 (PR), that originally proposed this solution. So this PR should either close that, or leave a TODO linking to that.